src/interface/file.rs

```diff
@@ -175,6 +180,30 @@ impl EmulatedFile {
 
         Ok(bytes_written)
     }
+
+    pub fn zero(&mut self, count: usize, offset: usize) -> std::io::Result<usize> {
```

Comment:
Maybe a more descriptive name if it's not going to be commented - fill_with_zerosat or something.